src/Compilers/Test/Utilities/CSharp/SemanticModelTestBase.cs

```diff
@@ -220,5 +219,41 @@ protected CompilationUtils.SemanticInfoSummary GetSemanticInfoForTest(string tes
         {
             return GetSemanticInfoForTest<ExpressionSyntax>(testSrc);
         }
+
+        protected string GetOperationTreeForTest<TSyntaxNode>(CSharpCompilation compilation)
+            where TSyntaxNode: SyntaxNode
+        {
+            var tree = compilation.SyntaxTrees[0];
+            var model = compilation.GetSemanticModel(tree);
+            SyntaxNode syntaxNode = GetSyntaxNodeOfTypeForBinding<TSyntaxNode>(GetSyntaxNodeList(tree));
```

Comment:
We will behave similar to the compiler binding tests for those cases. Seems that [this](http://source.roslyn.io/#Roslyn.Compilers.CSharp.Test.Utilities/SemanticModelTestBase.cs,22) method adds the nodes from bottom up, and picks the first node that matches full text and is of expected type. Also I am not sure if we can have 2 nodes of same kind with same span in tree.